MoskofidiAA/lab_5/main.cpp

```diff
@@ -0,0 +1,132 @@
+#include <iostream>
+#include <conio.h>
+
+using namespace std;
+const int N = 3;
+
+int multiply ( int m1[N][N], int m2[N][N])
```

Comment:
Function should only perform arithmetical computations and should not produce any output
Apply this fix to all others